backend/kind/up.sh

```diff
@@ -19,8 +19,13 @@ nodes:
   - containerPath: /code
     hostPath: ${APPLICATION_PATH}
     # readOnly: true
+networking:
+  disableDefaultCNI: true
 EOF
 
 kind create cluster --config kind-config.yaml --name=${CLUSTER_NAME}
 
+# TODO: Run this somewhere to enable weave network plugin:
+kubectl apply -f https://github.com/weaveworks/weave/releases/download/v2.6.2/weave-daemonset-k8s-1.11.yaml
```

Comment:
I wonder if this is ok here. I will run with a fresh cluster to see if I end up with a working weave pod.

Reply:
Works fine here. @mudler is there a better place to put this? Otherwise I will keep it here. This will be always applied no matter what you are planning to use your cluster for. Seems to work for my case here but I wonder if we want this in every use case.

Reply:
I don't think we can do much more, otherwise we need (yet another) flag to opt-in for this feature explictly. If doesn't hurt as default, I would keep as it is (better opinionated than giving too many options).

The only thing is, we need also to make the same change at the ekcp backend in Catapult. Note we can post also a kindconfig (is not documented anywhere.. my bad) https://github.com/mudler/ekcp/blob/b585bb2a22393f9b870651da06428917fdd6cf7c/api.go#L36 , so it should work somehow like this:

`curl -d "name=test&kindconfig=$(base64 kindconfig.yaml)" -X POST http://127.0.0.1:8030/new`